src/renderer.rs

```diff
@@ -530,7 +530,8 @@ impl Renderer {
 
         {
             // Present screen framebuffer to screen.
-            let mut p = f.pass(PassOp::Clear(Rgba::TRANSPARENT), present);
+            let background_appearance_color = Rgba::from(session.settings["background"].color());
```

Comment:
Since this variable is used on the next line only, and the value is clearly the "background color", I'd call it something much shorter like `bg`.

Reply:
I have since renamed it `bg`.